int32 object_id
bool is_visible
bool is_moving
float32 last_moved_robot_timestamp
float32 last_moved_start_robot_timestamp
float32 last_moved_time
float32 last_tapped_robot_timestamp
float32 last_tapped_time
float32 top_face_orientation_rad
Pose pose